src/style.css

```diff
@@ -0,0 +1,84 @@
+* {
+  padding: 0;
```

Comment:
avoid adding `padding` for all elements (f.e. div by default didn't have paddings)